commcare_connect/commcarehq/api.py

```
@@ -55,23 +57,48 @@ def get_case_list(api_key: HQApiKey, domain: str, filters: GetCaseDataAPIFilters
+    response = method(
         url,
         headers={"Authorization": f"ApiKey {api_key.user.email}:{api_key.api_key}"},
-        json=update_fields,
+        json=case_data,
     )
 
     try:
         response.raise_for_status()
     except httpx.HTTPStatusError as e:
-        raise CommCareHQAPIException(f"Failed to update case data for {domain} with {case_id}. HQ Error: {e}")
+        raise CommCareHQAPIException(f"{error_msg} HQ Error: {e}")
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)